Source/VMware.vSphereDSC/DSCResources/VMHostNetwork/VMHostNetworkMigration/VMHostVDSwitchMigration.ps1

```diff
@@ -187,6 +189,31 @@ class VMHostVDSwitchMigration : VMHostNetworkMigrationBaseDSC {
     <#
     .DESCRIPTION
 
+    Retrieves the Standard Port Group with the specified name if it exists.
+    Otherwise it throws an exception.
+    #>
+    [PSObject] GetStandardPortGroup($standardPortGroupName) {
+        try {
+            Write-VerboseLog -Message $this.RetrieveStandardPortGroupMessage -Arguments @($standardPortGroupName, $this.VMHost.Name)
```

Comment:
The **Write-Verbose** will need to be replaced with **WriteLogUtil** method call when the streams PR gets merged.